src/vs/workbench/contrib/notebook/browser/view/renderers/webviewPreloads.ts

```diff
@@ -1799,6 +1829,7 @@ async function webviewPreloads(ctx: PreloadContext) {
 	class OutputCell {
 
 		public readonly element: HTMLElement;
+		public readonly bottomElement: HTMLElement;
```

Comment:
Actually I don't need this change. Leftover from another idea I was trying out. I'll revert it.